service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/NotifyGatekeeperController.java

```diff
@@ -60,6 +64,12 @@ public void handleSubmittedEvent(@RequestBody CallbackRequest callbackRequest) {
         if (SUBMITTED.getValue().equals(callbackRequest.getCaseDetails().getState())) {
             applicationEventPublisher.publishEvent(new PopulateStandardDirectionsEvent(callbackRequest, requestData));
         }
-        applicationEventPublisher.publishEvent(new NotifyGatekeeperEvent(callbackRequest, requestData));
+        applicationEventPublisher.publishEvent(new NotifyGatekeepersEvent(callbackRequest, requestData));
+    }
+
+    private List<Element<EmailAddress>> resetGateKeeperEmailCollection() {
+        return List.of(
+            element(UUID.randomUUID(), EmailAddress.builder().email("").build())
```

Comment:
Do you need to set the email to an empty string here? What would happen if it is just null which is the case when .email isn't called?

Reply:
Setting to empty string expands the first item as in the collection on page load, as well as reseting the collection. It's not a requirement I guess to expand the collection page load but it's consistent with other events.